### PR TITLE
Updated Ticket Monster demo

### PR DESCRIPTION
**Pull request by sabre1041**
#### What does this PR do?

Updated the Ticket Monster Demo
#### How should this be manually tested?

Run through the demo as described in the PR
#### Is there a relevant Issue open for this?

No
#### Who would you like to review this?

/cc @etsauer

### CODE REVIEW

File: demos/ticket-monster/README.md [hunk 1]
```diff
@@ -1,119 +1,148 @@
 # Ticket-Monster OpenShift V3 Demo
 
-This folder contains the resources necessary to run the Ticket-Monster demonstration application on OpenShfit V3
+This demonstration describes the process for building and deploying Ticket-Monster  application on OpenShfit V3
 
-## What is Ticket-Monster?
+![Ticket Monster](images/ticketmonster.png "Ticket Monster")
 
-The Ticket-Monster is a moderately complex application that demonstrates how to build modern applications using JBoss web technologies. It is used to illustrate how to utilize JBoss Applications within the OpenShift V3 environment.
```
Comment (etsauer) [inline demos/ticket-monster/README.md]: @sabre1041 can you add a TOC? otherwise I think it's good.

**Comment by sabre1041** [comment]
@etsauer added table of contents to the demo. 

please review

File: demos/ticket-monster/README.md [hunk 2]
```diff
@@ -1,119 +1,160 @@
...
+## Table of Contents
 
-## Build and deployment Options
+* [Overview](#overview)
+* [Bill of Material](#bill-of-materials)
```
Comment (etsauer) [inline demos/ticket-monster/README.md]: Need an 's' at the end of Material

**Comment by sabre1041** [comment]
@etsauer corrected toc

**Pull request merged**